### PR TITLE
Add documentation for CLI 

### PR DESCRIPTION
**Pull request by mhmohona**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description
Added about CLI in documentation. 

### Related issue

Fixes - #152

### CODE REVIEW

**Comment by andrewtavis** [comment]
Is this ready for a final review, @mhmohona? 😊

**Comment by mhmohona** [comment]
Yes @andrewtavis. Its ready now.

File: docs/source/scribe_data/cli.rst [hunk 1]
```diff
@@ -1,6 +1,232 @@
...
+
+   .. code-block:: text
+
+      No data found for language 'english' and data type '['verbs']'.
+      Warning: No data file found for 'English' ['verbs']
+
+Notes:
+^^^^^^
+
+1. The command uses ``-l`` for language instead of ``-lang`` as previously documented. Both forms appear to be accepted.
```
Comment (andrewtavis) [inline docs/source/scribe_data/cli.rst:145]: I don't think we need to draw a distinction for this as this is the original release of it :) Not too many people have seen the original.

**Comment by andrewtavis** [comment]
Generally looks good, @mhmohona! I'll do some minor edits and then bring it in later today 😊

**Review by andrewtavis — APPROVED** [review]
Thanks so much for the hard work here, @mhmohona! Really coming along nicely, and it's so great that we'll have the docs to match the great functionality 😊

**Pull request merged**